measurementWriter.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import "fmt"
+
+type measurementWriter interface {
+	ReadMeasurement() error
+}
+
+type MeasurementPrintWriter struct {
+	mw          measurementWriter
```

Comment:
`mw` is not used at all. Additionally `measurementWriter` interface is not needed as well.